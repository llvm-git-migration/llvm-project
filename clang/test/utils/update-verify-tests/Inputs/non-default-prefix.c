// RUN: %clang_cc1 -verify=check %s
// RUN: diff %s %s.expected
void foo() {
    a = 2; // check-error{{asdf}}
           // expected-error@-1{ignored}}
}

