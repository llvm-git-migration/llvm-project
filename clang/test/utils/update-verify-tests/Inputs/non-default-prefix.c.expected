// RUN: %clang_cc1 -verify=check %s
// RUN: diff %s %s.expected
void foo() {
    a = 2; // check-error{{use of undeclared identifier 'a'}}
           // expected-error@-1{ignored}}
}

